src/lib/CanvasGroupings.js

```diff
@@ -0,0 +1,52 @@
+/**
+ *
+ */
+export default class CanvasGroupings {
+  /**
+   */
+  constructor(canvases, viewType = 'single') {
+    this.canvases = canvases;
+    this.viewType = viewType;
+    this._groupings = null; // eslint-disable-line no-underscore-dangle
+  }
+
+  /**
+   */
+  getCanvases(index) {
+    switch (this.viewType) {
+      case 'book':
+        return this.groupings()[Math.ceil(index / 2)];
+      default:
+        return this.groupings()[index];
+    }
+  }
+
+  /**
+   * Groups a set of canvases based on the view type. Single, is just an array
+   * of canvases, while book view creates pairs.
+   */
+  groupings() {
+    // if (this._groupings) { // eslint-disable-line no-underscore-dangle
+    //   return this._groupings; // eslint-disable-line no-underscore-dangle
+    // }
```

Comment:
Is this cruft or is this a `TODO` for later?

Reply:
Nice catch on that.. I've uncommented and pushed up.